packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -500,6 +509,7 @@ export class InputManager {
         }
         this._deviceSourceManager = new DeviceSourceManager(engine);
 
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars

```

Comment:
Why? If the variable is unused, just put an underscore in front of unused var to quiet.

Reply:
Fixed